opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryGuru.java

```diff
@@ -815,7 +815,14 @@ public void invalidateRepositories(Collection<? extends RepositoryInfo> repos, C
          * run in parallel to speed up the process.
          */
         final CountDownLatch latch = new CountDownLatch(repos.size());
-        final ExecutorService executor = Executors.newFixedThreadPool(env.getIndexingParallelism(),
+        int parallelismLevel;
+        // Both indexer and web app startup should be as quick as possible.
+        if (cmdType.equals(CommandTimeoutType.INDEXER) || cmdType.equals(CommandTimeoutType.WEBAPP_START)) {
```

Comment:
I think it's generally preferable to use `==` for enum comparison.

Reply:
I wondered about that when I was writing the comparisons.